services/schemas/soldier.zschema.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+
+const Soldier = z.object({
+    _id: z.string(),
```

Comment:
this is a uuid I think

Reply:
Don't think so -
`The id should be the soldier's private tt number.`